br/pkg/restore/split_client.go

```diff
@@ -64,7 +63,7 @@ type SplitClient interface {
 	ScatterRegions(ctx context.Context, regionInfo []*RegionInfo) error
 	// GetOperator gets the status of operator of the specified region.
 	GetOperator(ctx context.Context, regionID uint64) (*pdpb.GetOperatorResponse, error)
-	// ScanRegions gets a list of regions, starts from the region that contains key.
+	// ScanRegion gets a list of regions, starts from the region that contains key.
```

Comment:
🤔

Reply:
https://github.com/pingcap/tidb/pull/32876 fixed the typo, but this PR revert it.

Reply:
I have fixed it again.